dll/win32/msv1_0/msv1_0.c

```diff
@@ -1275,20 +1285,18 @@ LsaApLogonUserEx2(IN PLSA_CLIENT_REQUEST ClientRequest,
         if (LogonInfo->Password.Length)
         {
             // TODO: Check for Buffer limits wrt. ClientBufferBase and alignment.
-            LogonInfo->Password.Buffer = FIXUP_POINTER(LogonInfo->Password.Buffer, PtrOffset);
-            LogonInfo->Password.MaximumLength = LogonInfo->Password.Length;
+            LogonPass.Buffer = FIXUP_POINTER(LogonPass.Buffer, PtrOffset);
+            LogonPass.MaximumLength = LogonPass.Length;
         }
         else
         {
-            LogonInfo->Password.Buffer = NULL;
-            LogonInfo->Password.MaximumLength = 0;
+            LogonPass.Buffer = NULL;
+            LogonPass.MaximumLength = 0;
         }
 
-        TRACE("Domain: %S\n", LogonInfo->LogonDomainName.Buffer);
-        TRACE("User: %S\n", LogonInfo->UserName.Buffer);
-        TRACE("Password: %S\n", LogonInfo->Password.Buffer);
-
-        // TODO: If LogonType == Service, do some extra work using LogonInfo->Password.
```

Comment:
This TODO doesn't apply anymore?

Reply:
Sorry, my mistake. I restored this line.